### DISCUSSION TITLE
The commit log is extremely difficult to navigate

### DISCUSSION THREAD

**Discussion by nox** [servo/servo#26871] [General]
Between the full HTML template with the Markdown checklist and the dependabot extremely verbose HTML PR descriptions, 90% of my screen is stuff I don't care about when I run `git log`, and this is not helping when digging stuff. Is there any way we could improve that?

Cc @jdm

**Comment by jdm** [comment]
If it's possible to control the contents of the merge commit from homeu we could:
* Filter out anything from the dependabot user account
* Filter out the checklist contents
* Not include anything from the original PR description, just a link to the original PR

**Comment by nox** [comment]
I'm not sure I agree to three, I feel like human-written PR descriptions are useful, we could maybe filter everything below a first `--` or `<hr>` or something.

**Comment by jdm** [comment]
https://github.com/servo/homu/blob/d6b918f45b4aadfb43ae4cba5b9b80a4f9c00638/homu/main.py#L684-L689 is the code that creates the merge message.
